doc/nrf/getting_started/recommended_versions.rst

```diff
@@ -117,21 +117,21 @@ Other versions might also work, but are not verified.
          * - Tool
            - Toolchain Manager version
          * - Zephyr SDK
-           - |zephyr_sdk_recommended_ver_win10|
+           - :tool-version:`ZEPHYR_SDK_VERSION_WIN10`
```

Comment:
needs update, also commit messages with `ncs-...`

Reply:
Should be changed everywhere now